pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> A CLI to interact with HashiCorp Vault.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
```

Comment:
I'm a little wary of starting out with this much jargon from the get-go ("keyshares"? "unseal"?).

Would it be possible to start with a simpler example (say, just `vault init`, relying on the defaults for the parameters)?

Reply:
Using `vault init` would work, I'll simplify the example. This uses a default of 5 key shares requiring 3 of them to unseal.